hack/envtest/release-notes.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
+
+# This script is used to build the test binaries for the envtest package.
+if [ -z "${KUBERNETES_VERSION}" ]; then
+  echo "Missing KUBERNETES_VERSION environment variable"
+  exit 1
+fi
+
+# For each file in out/*.tar.gz, build a out/release-notes.md files containing a table
+# with every file and their respective hash
```

Comment:
nit: this doesn't seem entirely correct (we only generate on out/release-notes.md